lib/core/jhipster/reserved_keywords.js

```diff
@@ -77,6 +77,7 @@ const RESERVED_WORDS = {
     'VARINT', 'WHERE', 'WITH', 'WRITETIME', 'DISTINCT', 'BYTE', 'SMALLINT', 'COMPLEX', 'ENUM', 'DATE',
     'INTERVAL', 'MACADDR', 'BITSTRING'
   ],
+  COUCHBASE: [],
```

Comment:
Shouldn't values from [this]https://developer.couchbase.com/documentation/server/4.5/analytics/appendix_1_keywords.html) list be there and the following functions be updated?

Reply:
I tried to use these restricted words in a JHipster generated app (for example `all` and `document`) , I had no issue using them. Anyway, I'll add all this list of restricted words to prevent any error.